turbopack/crates/turbopack-node/src/evaluate.rs

```diff
@@ -214,12 +198,12 @@ pub async fn get_evaluate_pool(
     runtime_entries: Option<Vc<EvaluatableAssets>>,
     additional_invalidation: Vc<Completion>,
     debug: bool,
+    untracked_env_vars: bool,
```

Comment:
I would prefer a custom enum (`EnumVarsTracking::Tracked`/`EnumVarsTracking::Untracked`) here, because a bare `bool` is unclear when reading the callsites.

Alternatively, prefix each callsite's argument with a comment (`/* untracked_env_vars */ false,`)